packages/core/framework/src/http/types.ts

```diff
@@ -111,6 +111,20 @@ export type FileSystemRouteDescriptor = ScannedRouteDescriptor & {
   relativePath: string
 }
 
+export type ScannedMiddlewareDescriptor = {
+  matcher: string
+  method?: MiddlewareVerb | MiddlewareVerb[]
+  handler: MiddlewareFunction
+}
+
+export type BodyParserConfigRoutes = {
```

Comment:
suggestion: This type might need some adjustments based on previous comments

Reply:
Converted to an array instead